OutSystems/outsystems-datagrid-reactive
# RGRIDT-856: Added logic to enable set a column as not hidable

This PR is for adding a configuration to enable the developer to set a column as not hidable in run time by the end user.
Based on the [PR#90](https://github.com/OutSystems/outsystems-datagrid-reactive/pull/90) made by [@rmadeiraneto](https://github.com/rmadeiraneto).

### What was happening
* There was no lowcode way for the developer to make a column not hidable in runtime by the end-user.

### What was done
* Added a new boolean property to the ColumnConfig class
* Removed property not in use in ColumnConfig class
* Added formatItem callback to the ColumnPicker class where the new property will be checked

### Test Steps
1. Set the OptionalConfig CanBeHidden to false in a column
2. In runtime, open the Column Picker
2. Notice that the column will not be hidable


### Screenshots
![image](https://user-images.githubusercontent.com/10534623/122960390-081e0780-d38c-11eb-816d-e6f58ff951f3.png)


### Checklist
* [x] UML updated
